t/sanity.t

```diff
@@ -0,0 +1,89 @@
+use Test::Nginx::Socket::Lua 'no_plan';
```

Comment:
I'll try to remove this test file later.

I think we need to run `prometheus_test.lua` under `openresty` instead of the standard `Lua`.

This way we don't need to mock some of the share dict and ngx.log in openresty.

Reply:
If we run `prometheus_test.lua'with standard lua, then we can't use some features of openresty, such as `lua-resty-lrucache' this time. Performance will naturally be affected.